app/internal/page/task_list.go

```diff
@@ -6,6 +6,23 @@ import (
 	"github.com/ministryofjustice/opg-go-common/template"
 )
 
+const (
+	FillInLpaSection                      = "fillInTheLpa"
+	ProvideYourDetailsTask                = "provideYourDetails"
+	ChooseYourAttorneysTask               = "chooseYourAttorneys"
+	ChooseYourReplacementAttorneysTask    = "chooseYourReplacementAttorneys"
+	ChooseWhenTheLpaCanBeUsedTask         = "chooseWhenTheLpaCanBeUsed"
+	AddRestrictionsToLpaTask              = "addRestrictionsToTheLpa"
+	ChooseCertificateProviderTask         = "chooseYourCertificateProvider"
+	CheckAndSendToCertificateProviderTask = "checkAndSendToYourCertificateProvider"
+	PayForLpaSection                      = "payForTheLpa"
+	PayForTheLpaTask                      = "payForTheLpa"
+	ConfirmYourIdentityAndSignSection     = "confirmYourIdentityAndSign"
+	ConfirmYourIdentityAndSignTask        = "confirmYourIdentityAndSign"
+	RegisterTheLpaSection                 = "registerTheLpa"
+	RegisterTheLpaTask                    = "registerTheLpa"
+)
+
```

Comment:
Started this when I was considering grouping paths by section. Having worked through the ticket I don't think we need to (at the moment) but if we need to support a defined flow to be able to go back we could lean on the consts here in other places.